lib/pages/new_group/new_group.dart

```diff
@@ -131,6 +155,186 @@ class NewGroupController extends State<NewGroup>
+    );
+  }
+
+  void _handleUploadAvatarNewGroupChatOnData(BuildContext context, Either<Failure, Success> event) {
+    Logs().d('NewGroupController::_handleUploadAvatarNewGroupChatOnData()');
+    uploadAvatarNewGroupChatNotifier.value = event;
+    event.fold(
+      (failure) {
+        Logs().e('NewGroupController::_handleUploadAvatarNewGroupChatOnData() - failure: $failure');
+        removeAllImageSelected();
+      },
+      (success) {
+        Logs().d('NewGroupController::_handleUploadAvatarNewGroupChatOnData() - success: $success');
+        if (success is UploadContentSuccess) {
+          uriAvatar = success.uri;
+          removeAllImageSelected();
+        }
+      },
+    );
+  }
+
+  void _handleUploadAvatarNewGroupChatOnDone() {
+    Logs().d('NewGroupController::_handleUploadAvatarNewGroupChatOnDone() - done');
+  }
+
+  void _handleUploadAvatarNewGroupChatOnError(dynamic error, StackTrace? stackTrace) {
+    Logs().e('NewGroupController::_handleUploadAvatarNewGroupChatOnError() - error: $error | stackTrace: $stackTrace');
+  }
+
+  void _handleCreateNewGroupChatChatOnData(BuildContext context, Either<Failure, Success> event) {
+    Logs().d('NewGroupController::_handleCreateNewGroupChatChatOnData()');
+    createRoomStateNotifier.value = event;
+    event.fold(
+      (failure) {
+        Logs().e('NewGroupController::_handleCreateNewGroupChatChatOnData() - failure: $failure');
+      },
+      (success) {
+        Logs().d('NewGroupController::_handleCreateNewGroupChatChatOnData() - success: $success');
+        if (success is CreateNewGroupChatSuccess) {
+          removeAllImageSelected();
+          _goToRoom(context, success.roomId);
+        }
+      },
+    );
+  }
+
+  void _handleCreateNewGroupChatOnDone() {
+    Logs().d('NewGroupController::_handleCreateNewGroupChatOnDone() - done');
+  }
+
+  void _handleCreateNewGroupChatOnError(dynamic error, StackTrace? stackTrace) {
+    Logs().e('NewGroupController::_handleUploadAvatarNewGroupChatOnError() - error: $error | stackTrace: $stackTrace');
+  }
+
+  void _goToRoom(BuildContext context, String roomId) {
+    VRouter.of(context).toSegments(['rooms', roomId]);
+  }
+
+  Future<PermissionStatus>? _getCurrentPhotoPermission() {
+    return PermissionHandlerService().requestPermissionForPhotoActions();
+  }
+
+  Future<PermissionStatus>? _getCurrentCameraPermission() {
+    return PermissionHandlerService().requestPermissionForCameraActions();
+  }
+
+  void imagePickAction() async {
+    Navigator.pop(context);
+    final assetEntity = await CameraPicker.pickFromCamera(
+      context,
+      locale: window.locale,
+    );
+    if (assetEntity != null && assetEntity.type == AssetType.image) {
+      uploadAvatarNewGroupChat(
+        matrixClient: Matrix.of(context).client,
+        entity: imagePickerController.selectedAssets.first.asset
+      );
+    }
+  }
+
+  void _registerListenerForSelectedImagesChanged() {
+    imagePickerController.addListener(() {
```

Comment:
please make sure, user can not select more than one image in this case

Reply:
if (numberSelectedImagesNotifier.value == 1)

Yes, make sureee

Reply:
I mean, in the screen, what happen if user select more than one.